### PR TITLE
Wm 8 remove exec

### PR DESCRIPTION
**Pull request by EstebanDalelR**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1,318 +1,388 @@
...
-			owner = localowner;
-			
-			// Get issues by commit hash and owner
-			await octokit.request(`GET /search/issues?type=Commits`, {
-				org: owner,
-				q: `hash:${arrayOfSHAs[0]}`
-			}).then(octoresp => {
-				const issuesBySHAs = octoresp.data.items;
-
-				// Show toast if no search results are found
```
Comment (baristaGeek) [inline src/extension.ts:120]: Don't delete this. Please re-add it so we keep showing this toast for better UX.

File: src/extension.ts [hunk 2]
```diff
@@ -1,318 +1,388 @@
...
-					})
-				}
-			}).catch(error=> console.log("octoERR", error))
-
-		}
-	);
+function getPRsPerSHAs() {
+  octokit
+    .request(`GET /search/issues?type=Commits`, {
+      q: `hash:${arrayOfSHAs[0]}`,
```
Comment (baristaGeek) [inline src/extension.ts:132]: You're missing `org: owner,` so we don't get random pr comments from random repos in random organizations

**Review by baristaGeek — CHANGES_REQUESTED** [review]

**Comment by baristaGeek** [comment]
Also, please remember to remove console logs

File: src/extension.ts (hunk 2 above)
Comment (EstebanDalelR) [reply src/extension.ts:140]: this isn't in the dev branch yet

File: src/extension.ts (hunk 1 above)
Comment (EstebanDalelR) [reply src/extension.ts:120]: Readded

**Comment by baristaGeek** [comment]
You mentioned on Slack that retrieving org is out of scope. But it's not, we have it working on main right now and it's critical to not pull comments from unrelated repos from unrelated orgs. Hugs.

**Comment by EstebanDalelR** [comment]
> You mentioned on Slack that retrieving org is out of scope. But it's not, we have it working on main right now and it's critical to not pull comments from unrelated repos from unrelated orgs. Hugs.

Done, added back the line

**Review by baristaGeek — APPROVED** [review]
This looks amazing! This will give us a lot of stability. Ty @EstebanDalelR

**Pull request merged**